packages/shared/lib/wallet.ts

```diff
@@ -586,6 +584,28 @@ export const asyncCreateAccount = (): Promise<void> =>
         )
     })
 
+export const asyncRemoveWalletAccount = (accountId: string): Promise<void> =>
```

Comment:
These two API endpoint wrappers were added previously in `fix/profile-deletion` (#1209), but they're needed here for resetting the wallet state when the network switches.